Src/PerformanceCollector/Shared/Implementation/PerformanceCounterStructure.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    /// <summary>Represents basic performance counter structure.</summary>
```

Comment:
maybe. but it's not my invention - i just moved these fields a little bit - and it requires a lot of refactoring to get rid of it.